code/src/ThoughtsList.js

```diff
@@ -0,0 +1,40 @@
+import React, { useState } from 'react';
+
+import Moment from 'moment';
```

Comment:
`import moment from 'moment'` would be more reasonable, because `moment` is not a component, therefore we should import it with lowercase letter